### PR TITLE
Tickets/dm 42465: stuff needed for running hsc_weekley

### PR DESCRIPTION
**Pull request by eacharles**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -88,7 +88,7 @@ run-usdf-dev: export CM_DATABASE_URL=postgresql://cm-service@${CM_DATABASE_HOST}
 run-usdf-dev: export CM_ARQ_REDIS_URL=redis://${CM_ARQ_REDIS_HOST}:6379/1
 run-usdf-dev: export CM_DATABASE_PASSWORD=$(shell kubectl --cluster=usdf-cm-dev -n cm-service get secret/cm-pg-app -o jsonpath='{.data.password}' | base64 --decode)
 run-usdf-dev: export CM_ARQ_REDIS_PASSWORD=$(shell kubectl --cluster=usdf-cm-dev -n cm-service get secret/cm-redis-app -o jsonpath='{.data.password}' | base64 --decode)
-run-usdf-dev: export CM_DATABASE_ECHO=true
+run-usdf-dev: export CM_DATABASE_ECHO=True
```
Comment (fritzm) [inline Makefile:91]: Why?  I verified lower-case "true" works.  This is make syntax, not Python...

File: requirements/dev.txt [hunk 2]
```diff
@@ -85,9 +85,9 @@ coverage[toml]==7.4.1 \
     # via
     #   -r requirements/dev.in
     #   pytest-cov
-dill==0.3.7 \
-    --hash=sha256:76b122c08ef4ce2eedcd4d1abd8e641114bfc6c2867f49f3c41facf65bf19f5e \
-    --hash=sha256:cc1c8b182eb3013e24bd475ff2e9295af86c1a38eb1aff128dac8962a9ce3c03
+dill==0.3.8 \
```
Comment (fritzm) [inline requirements/dev.txt:88]: changes in requirements/ need to be collected/squashed onto a single commit, lest madness ensue...

File: src/lsst/cmservice/client/queues.py [hunk 3]
```diff
@@ -115,13 +115,20 @@ def pause_until_next_check(
...
         now = datetime.now()
-        print(now, sleep_time, wait_time, next_check)
+        try:
+            queue = self.get_row(row_id)
+            sleep_time = self.sleep_time(row_id)
+            wait_time = min(sleep_time, queue.interval)
+            delta_t = timedelta(seconds=wait_time)
+            next_check = queue.time_updated + delta_t
+        except Exception as msg:  # pylint: disable=broad-exception-caught
+            print(msg)
```
Comment (fritzm) [inline src/lsst/cmservice/client/queues.py:126]: should these `print` statements be logs instead?

File: src/lsst/cmservice/config.py [hunk 4]
```diff
@@ -23,6 +25,7 @@ class Configuration(BaseSettings):
     )
 
     database_password: str | None = Field(
+        default=None,
```
Comment (fritzm) [inline src/lsst/cmservice/config.py:28]: We are ping-ponging on this one, too.  It should not be permitted to run postgres/redis without these passwords specified.  What use case are you accommodating here so we can find a way to do that without allowing the running of unauthenticated servers?

**Review by fritzm — COMMENTED** [review]
Thanks, Eric!  Approved conditionally; a few questions, comments left.

Please separate requirements changes onto an isolated commit.

DM very highly discourages (some would say "prohibits") commits with "WIP", and "did blah", "more blah", "even more blah...", etc.  The (strong) recommendation is to squash these in a `rebase -i`, so e.g. one commit for "doc updates", one for "lint", one for "updated requiements", others for collected sets of related changes.  LMK if you would like me to take that on (you may notice I did this for all the x-mas surprise commits.)

File: Makefile (hunk 1 above)
Comment (eacharles) [reply Makefile:91]: It didn't work for me at some point, somewhere it was complaining that it had a string in stead of a boolean.   Maybe pydantic version dependent?

File: src/lsst/cmservice/client/queues.py (hunk 3 above)
Comment (eacharles) [reply src/lsst/cmservice/client/queues.py:126]: Do we have a logger?

File: src/lsst/cmservice/config.py (hunk 4 above)
Comment (eacharles) [reply src/lsst/cmservice/config.py:28]: I think the issue was that the syntax changed 2/ pydatntic 
and this was my effort to get things to run...

File: requirements/dev.txt (hunk 2 above)
Comment (eacharles) [reply requirements/dev.txt:88]: I don't know how to do this.

**Pull request merged**